t/rose-host-select/02-localhost.t

```diff
@@ -65,7 +65,7 @@ if [[ -n ${MORE_HOST} ]]; then
     sed -i 's/env CYLC_VERSION=[^ ]* //' "${TEST_KEY_BASE}.out.${MORE_HOST}"
     file_cmp "${TEST_KEY_BASE}.out.${MORE_HOST}" \
         "${TEST_KEY_BASE}.out.${MORE_HOST}" <<__OUT__
-[INFO] YYYY-MM-DDTHHMM ssh -oBatchMode=yes -oConnectTimeout=10 ${MORE_HOST} rose host-select-client <<'__STDIN__'
+[INFO] YYYY-MM-DDTHHMM ssh -oBatchMode=yes -oConnectTimeout=10 ${MORE_HOST} 'bash -l -c '"'"'rose host-select-client'"'"'' <<'__STDIN__'
```

Comment:
Command format has changed.

Reply:
This fails if you have `CYLC_ENV_NAME` set.